buildSrc/src/main/groovy/io.deephaven.java-common-conventions.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id 'base'
+    id 'java'
+    id 'io.deephaven.java-repository-conventions'
+    id 'io.deephaven.java-classpath-conventions'
+    id 'io.deephaven.java-license-conventions'
+    id 'io.deephaven.java-header-conventions'
+    id 'io.deephaven.java-toolchain-conventions'
+    id 'io.deephaven.java-jar-conventions'
+}
+
+project.tasks.getByName('quick').dependsOn project.tasks.withType(JavaCompile)
+
+configurations.all({ c ->
+    // Make dynamic versions illegal.
+    c.dependencies.all({
+        Dependency dep ->
+            if (dep.version && dep.version.endsWith('+')) {
+                throw new GradleException("Dynamic versions not allowed ($dep found in $path)")
+            }
+    })
+    c.resolutionStrategy {
+        // by default, we don't use any changing modules.
+        // however, if a developer is working on something where jars
+        // are published to a local filesystem (like mavenLocal()),
```

Comment:
I think I was wrong on this original comment: filesystem repos are _always_ "rechecked" b/c they are just files on disk.

Reply:
Does this mean that resolutionStrategy closure should be deleted?